en/tutorials/model-configuration/ipex-llm.md

```diff
@@ -0,0 +1,15 @@
+# Integrating Local Models Deployed with IPEX-LLM
+
+[IPEX-LLM](https://github.com/intel-analytics/ipex-llm) is a PyTorch library for running LLM on Intel CPU and GPU (e.g., local PC with iGPU, discrete GPU such as Arc, Flex and Max) with very low latency. 

```

Comment:
Arc -> Arc A-Series ?

As MTL iGPU is also named as Intel Arc Graphics

Reply:
I used the same sentence as in IPEX-LLM README